micronutrientsupport/micronutrient-support-tool
# Dw compound selection persist

- Stores selected mn compounds for each intervention in local storage. 

To Test: 
- Variables: Malawi & Calcium
- Load 2 interventions (ID 4 & 5)
- Select as many Mn as you like in stage 1
- Select compound for each mn
- Move between stages 1-3
- Repeat for other intervention
- Review initial intervention and make sure selected micronutrients and compounds are selected
